SMRefLocation/schema-LD.json

```diff
@@ -0,0 +1,39 @@
+{

```

Comment:
possibly not necessary. see contribution manual.

Reply:
This might be true, see my previous replies.